src/utils/uiUtils.ts

```diff
@@ -64,4 +74,39 @@ export class UIUtils {
             quickPick.show();
         });
     }
+
+    static getWorkspaceDir(): string {
```

Comment:
Sorry, to be clear: I think this method, `STATIC_RESOURCES_PATH` above, and `getStaticResourcesDir()` below should also end up in `workspaceUtils`, as none of those are particularly related to "UI". Can we move these over as well?